app/models/ncr_dispatcher.rb

```diff
@@ -1,67 +0,0 @@
-class NcrDispatcher < LinearDispatcher
```

Comment:
ah I see you are getting rid of the subclass.

My fear is that if we have 100 clients, our Dispatcher is a long list of `if proposal.client_slug == whatever` chains.